includes/admin/reporting/reports.php

```diff
@@ -2548,377 +2547,454 @@ function display_export_report() {
-								<div class="edd-from-to-wrapper">
+	?>
+	<div id="edd-dashboard-widgets-wrap">
+		<div class="metabox-holder">
+			<div id="post-body">
+				<div id="post-body-content" class="edd-reports-export edd-admin--has-grid">
+
+				<?php do_action( 'edd_reports_tab_export_content_top' ); ?>
+
+				<div class="postbox edd-export-earnings-report">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Earnings Report', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV giving a detailed look into earnings over time.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-earnings" class="edd-export-form edd-import-export-form" method="post">
+							<div class="edd-to-and-from-container">
+							<label for="start_month" class="screen-reader-text">Start Month</label>
+								<?php echo EDD()->html->month_dropdown( 'start_month' ); ?>
+							<label for="start_year" class="screen-reader-text">Start Year</label>
+								<?php echo EDD()->html->year_dropdown( 'start_year' ); ?>
+							</div>
+
+							<span class="edd-to-and-from--separator"><?php echo _x( '&mdash; to &mdash;', 'Date one to date two', 'easy-digital-downloads' ); ?></span>
+
+							<div class="edd-to-and-from-container">
+							<label for="end_month" class="screen-reader-text">End Month</label>
+								<?php echo EDD()->html->month_dropdown( 'end_month' ); ?>
+							<label for="end_year" class="screen-reader-text">End Year</label>
+								<?php echo EDD()->html->year_dropdown( 'end_year' ); ?>
+							</div>
+							<?php wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' ); ?>
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_Earnings_Report_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-sales-earnings">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Sales and Earnings', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV of all sales or earnings on a day-by-day basis.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-sales-earnings" class="edd-export-form edd-import-export-form" method="post">
+							<div class="edd-from-to-wrapper">
+								<label for="edd-order-export-start" class="screen-reader-text">Export Sales and Earnings Start Date</label>
 									<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-order-export-start',
+											'class'       => 'edd-export-start',
+											'name'        => 'start',
+											'placeholder' => _x( 'From', 'date filter', 'easy-digital-downloads' ),
+										)
+									);
+									?>
+								<label for="edd-order-export-end" class="screen-reader-text">Export Sales and Earnings End Date</label>
```

Comment:
This label text (and all following) should still be in `__()` so it can be translated please.

Reply:
I totally blanked on that.  Thanks.